### PR TITLE
feat: add keyboard shortcut

### PR DESCRIPTION
**Pull request by MegaBluejay**
I don't like cluttering up my toolbar with too many extension icons, so keyboard shortcuts for little ones like this are quite useful imo.

Maybe the script call is worth extracting into a function since it's repeated a few times, but I didn't want to take it upon myself to refactor.

### CODE REVIEW

**Comment by ansh** [comment]
Nice shortcut! 

Any reason for choosing Alt-Shift-B? @MegaBluejay

**Comment by MegaBluejay** [comment]
> Nice shortcut!
> 
> Any reason for choosing Alt-Shift-B? @MegaBluejay

Not really. I took Alt-Shift from dark reader's default settings, but I haven't seen it anywhere else, so it seems like a good combo to not clash with browser/os bindings
B is just mnemonic for bionic

**Comment by milkyomo** [comment]
I think there's a typo on line 16 in the manifest.json: "suggested_key" instead of "suggensted_key"

File: manifest.json [hunk 1]
```diff
@@ -10,5 +10,13 @@
   "host_permissions": ["https://*/*"],
   "action": {
     "default_popup": "src/popup.html"
+  },
+  "commands": {
+    "toggle-bionic": {
+      "suggensted_key": {
+        "default": "Alt-Shift-B"
```
Comment (X140Yu) [inline manifest.json:17]:
```suggestion
      "suggested_key": {
        "default": "Alt+Shift+B"
```

**Comment by X140Yu** [comment]
Nice, we can not set a shortcut in Chrome Extension Settings

image

**Pull request merged**